src/core/api/clock.ts

```diff
@@ -87,6 +87,11 @@ export interface IStalledStatus {
            "buffering"; // Other cases
   /** `performance.now` at the time the stalling happened. */
   timestamp : number;
+  /**
+   * Position, in seconds, at which data is awaited.
```

Comment:
Is that really correct ?
You define `stalledPosition = currentTime + bufferGap;`, which seems to be the position where we want to seek later, instead of the position at which data is awaited.

Reply:
I'm under the impression that `bufferGap` works the following way:
  - if we are currently playing buffered data, it's the different between the end of the current range and `currentTime`
  - if we are currently inside an unbuffered rage, it is set to `Infinity` (makes no much sense to me but that's legacy and I'm afraid of breaking things)

So technically `currentTime + bufferGap` when `bufferGap is finite is equivalent to `currentRange.end`. But I found that it made much sense in that function to re-perform the operation in the other way, as it uses predominantly those and has no idea of the equivalence between `currentRange` and `bufferGap`.